h/templates/panels/navbar.html.jinja2

```diff
@@ -13,6 +13,20 @@
             id="search-bar"
             action="{{ search_url }}">
         {{ svg_icon('search', 'search-bar__icon') }}
+        <div class="search-bar__lozenges">
+          {% if opts['search_username'] %}
```

Comment:
Unless there is any reason not to @sheetaluk, I am going to move this logic of iterating and adding lozenges into the .search-bar__lozenges container below. It has the `data-ref="searchBarLozenges"` that is used in the search controller and tracked with this._lozengeContainer.